server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -251,6 +251,11 @@ it is around the label */
     @include u-padding-top(4);
     @include u-padding-bottom(2);
   }
+
+  // Hide the government name on desktop (if set to hidden)
+  .cf-hidden-gov-name {
```

Comment:
suggestion (non-blocking): I'd prefer to call this `cf-gov-name` since it's not always hidden.

Reply:
The confusing thing is that this element actually doesn't exist unless they have the flag turned on to hide the government name. But the reason governments may want to hide the name is if the logo has the name in it and with northstar, the logo disappears on mobile, so this is basically to show the hidden name if the logo is hidden (on mobile) and to hide it on desktop. Thoughts on the naming based on that? `cf-gov-name-with-hidden-flag` seems too long but something like that may be good? Or maybe `cf-gov-name` still makes sense but wanted to make sure it was clear first. 

https://github.com/user-attachments/assets/4c285938-3718-4e2c-b33a-4b696f6532e7

Reply:
Yeah, I like that idea - updated!

Reply:
Instead of creating custom classes can you use the `.display-none` / `.tablet:display-block` USWDS [utility classes](https://designsystem.digital.gov/utilities/display)?

Reply:
Good call! I had followed the pattern of cf-header-logo, but makes sense to use the utility class here. Updated - kept the custom class just for the test.